site/tsconfig.json

```diff
@@ -12,16 +13,14 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "es5",
-    "typeRoots": ["./src/types", "node_modules/@types"]
+    "target": "es5"
   },
   "include": [
-    "./next-env.d.ts",
-    "./src/**/*.ts",
-    "./src/**/*.tsx",
-    "theme-override.d.ts",
     "assets.d.ts",
-    "./scripts/**/*.ts"
+    "next-env.d.ts",
+    "theme-override.d.ts",
+    "**/*.ts",
+    "**/*.tsx"
   ],
-  "exclude": ["node_modules", ".next"]
+  "exclude": ["node_modules"]
 }
```

Comment:
I made the above changes by creating a new Next app from scratch and looking at its config. We don't need `"typeRoots"` – this config was borrowed from HASH frontend.